### PR TITLE
[DR-2524] Create Olivia's datarepo Helm definition

### PR DESCRIPTION
**Pull request by okotsopoulos**
- https://broadworkbench.atlassian.net/browse/DR-2524

I am following step 8.2 in our ["Jade Getting Started"](https://github.com/DataBiosphere/jade-data-repo/blob/develop/docs/jade-getting-started.md) guide to create my datarepo Helm definition.

### CODE REVIEW

File: dev/ok/helmfile.yaml
Comment (snf2ye) [inline dev/ok/helmfile.yaml:45]: latest = 0.0.31
https://github.com/broadinstitute/datarepo-helm/blob/master/charts/datarepo/Chart.lock#L7

File: dev/ok/datarepo-api.yaml
Comment (okotsopoulos) [reply dev/ok/datarepo-api.yaml]: Appreciate this context!  We ran into this kind of disconnection when removing env variables from WFL too :)

File: dev/ok/gcloud-sqlproxy.yaml
Comment (okotsopoulos) [reply dev/ok/gcloud-sqlproxy.yaml]: Thanks for the link!

File: dev/ok/helmfile.yaml
Comment (okotsopoulos) [reply dev/ok/helmfile.yaml]: Updated.  I can make a change to the onboarding docs indicating that we should check our new helmfile against the latest chart versions.  Is the reason that `ok-jade-datarepo-api` and `ok-jade-datarepo-ui` don't have versions specified here so that they always pick up latest?

Comment (snf2ye) [reply dev/ok/helmfile.yaml]: Perfect and exactly!

**Review by snf2ye — APPROVED** [review]

**Comment by okotsopoulos** [comment]
As I have the requisite approvals, I'll merge this PR and then work through the [GCP set-up](https://github.com/DataBiosphere/jade-data-repo/blob/develop/docs/jade-getting-started.md#9-google-cloud-platform-setup) to bring up my Helm services.

**Comment by okotsopoulos** [comment]
Helmfile changes successfully applied after merge via [onboarding instructions](https://github.com/DataBiosphere/jade-data-repo/blob/develop/docs/jade-getting-started.md#9-google-cloud-platform-setup):

```
cd ~/datarepo-helm-definitions/dev/ok
git checkout master
git pull --rebase

gcloud container clusters get-credentials dev-master --region us-central1 --project broad-jade-dev

helmfile apply
```

My namespaces exist:
https://console.cloud.google.com/kubernetes/discovery?project=broad-jade-dev&pageState=(%22savedViews%22:(%22i%22:%22422f4fc695cb4c7d8310e887c10ca448%22,%22c%22:%5B%5D,%22n%22:%5B%22ok%22%5D))

**Pull request merged**
